hare/clock_test.go

```diff
@@ -28,14 +26,10 @@ func TestClock_AwaitWakeup(t *testing.T) {
 }
 
 func TestClock_AwaitRound(t *testing.T) {
-	if util.IsWindows() && util.IsCi() {
-		t.Skip("Skipping test in Windows on CI (https://github.com/spacemeshos/go-spacemesh/issues/3628)")
-	}
-
 	r := require.New(t)
 
-	tolerance := 5 * time.Millisecond
-	waitTime := 20 * time.Millisecond
+	tolerance := 20 * time.Millisecond
+	waitTime := 80 * time.Millisecond
```

Comment:
do you know why this is needed?

Reply:
My assumption is because the windows runner is so much slower than the ubuntu and macos runner 25 ms is just not enough time for the timer that is used to await the round to tick once.

`AwaitEndOfRound` uses `time.AfterFunc` which internally uses `time.Timer`. According to the documentation it "will send the current time on its channel after at least duration d". Essentially that means the timer will never be early, but there are no guarantees about delays.

So `<-time.After(1 * time.Second)` will fire earliest in 1 second, but could also fire later (on a busy / slow system).

The assertions in this test compare `AwaitEndOfRound` to `time.After` which has the same issue (as in it can never be early but it can be late). So the test becomes less stable the slower / busier the system is it is running on.